app/src/main/java/com/example/triptracker/view/home/HomeUtils.kt

```diff
@@ -353,10 +357,7 @@ fun DisplayItinerary(
                               color = md_theme_light_onPrimary,
                               modifier = Modifier.testTag("Title"))
                           Text(
-                              text =
-                                  if (ambientProfile.userProfile.value.flowerMode == 1)
-                                      "${itinerary.flameCount} ${flowerStringBasedOnCount(itinerary.flameCount)}"
-                                  else "${itinerary.flameCount} 🔥",
+                              text = getText(ambientProfile, itinerary),
```

Comment:
Maybe you could rename the function to something like getTrendingCount to make it easier to understand

Reply:
Done ! thanks for the comment !